lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -83,6 +85,9 @@ sub _add_routes {
         if $op_spec->{operationId} and $uniq{o}{$op_spec->{operationId}}++;
       die qq([OpenAPI] Route name "$name" is not unique.) if $name and $uniq{r}{$name}++;
 
+      if ($op_spec->{security}) {
+        $route = $route->under('/')->to(cb => $self->_security_action($op_spec->{security}));
```

Comment:
The per-op under can't replace the global route. That would cause the security definitions to build up as you generate more routes!